amf-api-contract/shared/src/main/scala/amf/apicontract/internal/spec/async/parser/domain/Async20ServerVariableParser.scala

```diff
@@ -0,0 +1,107 @@
+package amf.apicontract.internal.spec.async.parser.domain
+
+import amf.apicontract.client.scala.model.domain.{Parameter, Server}
+import amf.apicontract.internal.metamodel.domain.{ParameterModel, ServerModel}
+import amf.apicontract.internal.spec.async.parser.context.AsyncWebApiContext
+import amf.apicontract.internal.spec.common.WebApiDeclarations.{ErrorServer, ErrorServerVariable}
+import amf.apicontract.internal.spec.oas.parser.domain.OasLikeServerVariableParser
+import amf.apicontract.internal.spec.spec.OasDefinitions
+import amf.core.client.scala.model.domain.{AmfArray, AmfScalar}
+import amf.core.internal.metamodel.domain.ExternalSourceElementModel
+import amf.core.internal.parser.YMapOps
+import amf.core.internal.parser.domain.{Annotations, ScalarNode, SearchScope}
+import amf.core.internal.utils.IdCounter
+import amf.core.internal.validation.CoreValidations
+import amf.shapes.client.scala.model.domain.Example
+import amf.shapes.internal.domain.metamodel.common.ExamplesField
+import amf.shapes.internal.spec.common.parser.YMapEntryLike
+import amf.shapes.internal.validation.definitions.ShapeParserSideValidations.ExamplesMustBeASeq
+import org.yaml.model.{YMap, YMapEntry, YNode, YSequence, YType}
+
+case class Async20ServerVariableParser(entry: YMapEntryLike, parent: String)(implicit override val ctx: AsyncWebApiContext)
```

Comment:
The name change in the class is ok, but revert the name change in the file

Reply:
done